db/controller/controller.js

```diff
@@ -27,8 +28,18 @@ exports.getArticlesById = (req, res, next) => {
     });
 };
 
+exports.getCommentsById = (req, res, next) => {
```

Comment:
You could specify that you're getting the comments by their article ID (just so they don't think you're using their comment id), for readability.